xml/System.Runtime.InteropServices/DllImportSearchPath.xml

```diff
@@ -137,7 +137,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>When searching for assembly dependencies, include the directory that contains the assembly itself, and search that directory first. This value is used by the .NET Framework, before the paths are passed to the Win32 <see href="https://go.microsoft.com/fwlink/?LinkId=236091">LoadLibraryEx</see> function.</summary>
+        <summary>When searching for assembly dependencies, include the directory that contains the assembly itself, and search that directory first. When used in Native AOT and single-file scenarios, the application's installation directory is considered the "assembly directory" and is searched.</summary>
```

Comment:
```suggestion
        <summary>When searching for assembly dependencies, include the directory that contains the assembly itself, and search that directory first. When used in Native AOT and single-file deployment models, the application's installation directory is considered the "assembly directory" and is searched.</summary>
```
To match how we call them in other places in the docs (https://learn.microsoft.com/en-us/dotnet/core/deploying/).